### ISSUE TITLE
Add a util module that gives named functions on Integer, Float, and BigDecimal

### ISSUE THREAD

**Issue by thegedge** [Shopify/measured#39]
For example, `1.metres`.

**Comment by kmcphillips** [comment]
Few points about this. If we decide we want to:

* Look at how `ActiveSupport` does times for a [reference implementation](https://github.com/rails/rails/blob/92703a9ea5d8b96f30e0b706b801c9185ef14f0e/activesupport/lib/active_support/core_ext/numeric/time.rb#L29-L32). 
* We would probably need to define these via the measurement DSL. So [when we define](https://github.com/Shopify/measured/blob/master/lib/measured/units/length.rb) the units, we'd need to create the methods (singular and plural) on the numeric base classes.
* This should _not_ be on by default. Should be opt in. You can also look at how measurements are [split out from the base library implementation](https://github.com/Shopify/measured/blob/2e26bc56959cfb936c056b8f69fc169e93d5edac/lib/measured.rb).

One of the major reasons we built rather than used was that so many existing tools overrode core types and added methods globally. That's dangerous behaviour, unless you opt into wanting it.

**Comment by thegedge** [comment]
Yep, I really should have mentioned that I intended for this to be opt-in, similar to `bigdecimal/util` adding `to_d` to a bunch of core types. Thanks for calling that out!

**Comment by thegedge** [comment]
@kmcphillips what do you think of having `Measured.build(add_numeric_extensions: true)` to support this? Or do you think it would be better to have something like `require 'measured/numeric_extensions'` that wraps `Measured.build`? I'm thinking something like the latter, because it would be the only way for us to wrap `Length` / `Weight`.

**Comment by kmcphillips** [comment]
This doesn't need to be part of the DSL. I'd much rather it be an optional, not included by default, extension that you can require.
